sw/airborne/math/pprz_algebra_float.h

```diff
@@ -721,6 +767,58 @@ static inline void float_mat_vect_mul(float *o, float **a, float *b, int m, int
   }
 }
 
+/** a *= k, where k is a scalar value
+ *
+ * a: [m x n]
+ * k: [1 x 1]
+ */
+static inline void float_mat_scale(float **a, float k, int m, int n)
+{
+  int i, j;
+  for (i = 0; i < m; i++) {
+    for (j = 0; j < n; j++) {
+      if (a[i][j] != 0.0) {
+        a[i][j] *= k;
+      }
+    }
+  }
+}
+
+/** a += k*b, where k is a scalar value
+ *
+ * a: [m x n]
+ * b: [m x n]
+ * k: [1 x 1]
+ */
+static inline void float_mat_sum_scaled(float **a, float **b, float k, int m, int n)
+{
+  int i, j;
+  for (i = 0; i < m; i++) {
+    for (j = 0; j < n; j++) {
+      if (a[i][j] != 0.0) {
+        a[i][j] += k * b[i][j];
+      }
+    }
+  }
+}
+
+
+/** a += k*b, where k is a scalar value
```

Comment:
The description here is the same as above. The code looks the same too...

Reply:
See answer above.